examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+        "    cmd = [\"sox\", sph_path, wav_path]\n",
+        "    subprocess.run(cmd)\n",
+        "print(\"Finished conversion.\\n******\")"
+      ],
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "/content\n",
+            "******\n",
+            "Dataset downloaded at: data/an4_sphere.tar.gz\n",
+            "Converting .sph to .wav...\n",
+            "Finished conversion.\n",
+            "******\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LEKDkOSimsKr",
+        "colab_type": "text"
+      },
+      "source": [
+        "Since an4 is not designed for speaker recognition, this facilitates the oppurtunity to demostrate how you can generate manifest files that are necessary for training. These methods can be applied to any dataset to get similar training manifest files. \n",
+        "\n",
+        "First get a scp file(s) which has all the wav files with absolute path for each of train,dev and test set. This can be easily done by `find` bash command"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "0e6nuOFN8Pfv",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!find $PWD/data/an4/wav/an4_clstk  -iname \"*.wav\" > data/an4/wav/an4_clstk/train_all.scp"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7168Z9eXn4st",
+        "colab_type": "text"
+      },
+      "source": [
+        "Let's look at first 3 lines of scp file for train. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "SQupCVpZIvtL",
+        "colab_type": "code",
+        "outputId": "e45cf645-42fc-4f4f-bd94-964848e04145",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 68
+        }
+      },
+      "source": [
+        "!head -n 3 data/an4/wav/an4_clstk/train_all.scp"
+      ],
+      "execution_count": 4,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "/content/data/an4/wav/an4_clstk/fsrb/an169-fsrb-b.wav\n",
+            "/content/data/an4/wav/an4_clstk/fsrb/an166-fsrb-b.wav\n",
+            "/content/data/an4/wav/an4_clstk/fsrb/an170-fsrb-b.wav\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cN09z0XFoDjN",
+        "colab_type": "text"
+      },
+      "source": [
+        "Since we created scp file for train, we use `scp_to_manifest.py` to convert this scp file to manifest and then optionally split the files to train \\& dev for evaluating the models while training by using `--split` flag. So as you guessed we wouldn't be needing `--split` option for test folder. "
```

Comment:
the scp file.
to a manifest
\& shows up incorrectly
the --split flag
the --split option

Reply:
Done